src/torchcodec/_core/FFMPEGCommon.cpp

```diff
@@ -56,6 +56,46 @@ int64_t getDuration(const UniqueAVFrame& avFrame) {
 #endif
 }
 
+const int* getSupportedSampleRates(const AVCodec& avCodec) {
+  const int* supportedSampleRates = nullptr;
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(61, 13, 100)
+  int numSampleRates = 0;
+  int ret = avcodec_get_supported_config(
+      nullptr,
+      &avCodec,
+      AV_CODEC_CONFIG_SAMPLE_RATE,
+      0,
+      (const void**)&supportedSampleRates,
```

Comment:
Let's do a `reinterpret_cast` here; https://en.cppreference.com/w/cpp/language/reinterpret_cast.html. In general, we should never do a C-style cast, as it's not always obvious what _kind_ of cast a C-style cast will actually become: https://en.cppreference.com/w/cpp/language/explicit_cast.html